docs/guides/qiskit-addons-mpf-get-started.ipynb

```diff
@@ -170,6 +170,14 @@
       "Exact solution: [ 0.17239057 -1.19447005  2.02207947]\n",
       "Approximate solution: [-0.40454257  0.57553173  0.8290123 ]\n"
      ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/runner/work/documentation/documentation/.tox/py311/lib/python3.11/site-packages/osqp/interface.py:405: PendingDeprecationWarning: The default value of raise_error will change to True in the future.\n",
+      "  warnings.warn(\n"
+     ]
```

Comment:
Maybe should manually remove this? Users won't usually see it but I don't like the principle of editing outputs.